engine/src/main/java/org/terasology/rendering/nui/widgets/UIDropdown.java

```diff
@@ -20,11 +20,7 @@
 import org.terasology.math.geom.Rect2i;
 import org.terasology.math.geom.Vector2i;
 import org.terasology.rendering.assets.font.Font;
-import org.terasology.rendering.nui.BaseInteractionListener;
-import org.terasology.rendering.nui.Canvas;
-import org.terasology.rendering.nui.CoreWidget;
-import org.terasology.rendering.nui.InteractionListener;
-import org.terasology.rendering.nui.SubRegion;
+import org.terasology.rendering.nui.*;
```

Comment:
No wildcard imports, please! (Set min. # of imports before wildcards are used to 999 in IDEA settings)